apps/portal/src/common/constants/notification.ts

```diff
@@ -19,6 +19,8 @@ export const ChannelType = {
   SMS_KAPSYSTEM: 8,
   PUSH_SNS: 9,
   VC_TWILIO: 10,
+  AWS_SES: 11,
```

Comment:
Are these values new?
Are these something we merged in backend too?

Reply:
SNS_SMS changes are merged
AWS_SES is yet to be merged